python/lsst/validate/drp/validate.py

```diff
@@ -97,7 +97,8 @@ def get_filter_name_from_job(job):
     return filter_name
 
 
-def run(repo_or_json, metrics=None, makePrint=True, makePlot=True,
+def run(repo_or_json, metrics=None,
+        outputPrefix=None, makePrint=True, makePlot=True,
```

Comment:
Correct, it is intentionally and specifically different.  A user should be allowed to pass `outputPrefix=''` to mean don't prepend anything.  But if the default of `outputPrefix=None` is used, then it automatically computes an appropriate prefix based on the repo name.